arrayMethod.js

```diff
@@ -0,0 +1,128 @@
+//1
+const plus10 = (arr) => {
+  return arr.map((el) => {
+    return el + 10
+  })
+}
+console.log(plus10([1,2,3]));
+
+//2
+
+const emptyStr = (arr) => {
+  return arr.map((el) =>{
+      if(typeof el === "string" ){
+      return el
+    }else{
+      return ""
+    }
+  })
+  }
+//}
+console.log(emptyStr(["1", "b", true, {name: "cat"}]));
+
+//3
+
+const evenNum = (arr) => {
+  return arr.filter((el) => {
+    return el % 2 === 0;
+  })
+}
+console.log(evenNum([1,2,3,4,5]));
+
+//4
+const sameArray = (arr) => {
+  return arr.every((el) => {
+    if (el === arr[el]){
```

Comment:
This wouldn't work because you're target think the array at the element. The only reason it worked is because you were testing with an array of ones. so arr[el] is === to arr[1] which will compare all to that element at that index. But lets imagine you got a string instead of a number. Long story short, arr[0] would be much better at this case. :)